backend/README.md

```diff
@@ -20,6 +20,18 @@ For example, when running locally, visit:
 http://localhost:8080/swagger-ui.html
 ```
 
+### myShifts endpoint
+
+Returns shifts of a given user (inferred from the authentication token).
+At the moment authentication has not been implemented yet, so the "token" is just user ID.
+Example call:
+` curl http://localhost:8080/myShifts?start=1410-07-15T00:00:00Z&2100-01-01T00:00:00Z -H
+ 'Authorization: 1' ` 
```

Comment:
should there be an `end=` in the request URL? (after `&`).  also when I run the command I get `bash: -H command not found`

Reply:
Yes, there should be an `end=`
The error is weird, -H is a curl flag not a bash command. Are you running everything on one line?

Reply:
Yup 
![image](https://user-images.githubusercontent.com/20900854/79973883-3e15e200-8490-11ea-9ea5-008cb7669bb0.png)

Reply:
Solved it, need quotes around the request, probably dependent on the shell